spec.emu

```diff
@@ -275,12 +276,97 @@
       <emu-note>That means that "A" (`LATIN CAPITAL LETTER A`) measures as 1 code unit, and "🔥" (`FIRE`) measures as 2 code units. Source maps for other content types may diverge from this.</emu-note>
     </dd>
   </dl>
+
+  <emu-clause id="sec-vlq-grammar">
+    <h1>base64 VLQ grammar</h1>
+    <p>A base64-encoded variable-length quantity adheres to the following lexical grammar:</p>
+
+    <emu-grammar type="definition">
+      Vlq ::
+        VlqDigitList
+
+      VlqDigitList ::
+        TerminalDigit
+        ContinuationDigit VlqDigitList
+
+      TerminalDigit ::
+        OneOfTerminalDigit
+
+      ContinuationDigit ::
+        OneOfContinuationDigit
+
+      OneOfTerminalDigit :: one of
+        `A` `B` `C` `D` `E` `F` `G` `H` `I` `J` `K` `L` `M` `N` `O` `P` `Q` `R`
+        `S` `T` `U` `V` `W` `X` `Y` `Z` `a` `b` `c` `d` `e` `f`
+
+      OneOfContinuationDigit :: one of
+        `g` `h` `i` `j` `k` `l` `m` `n` `o` `p` `q` `r` `s` `t` `u` `v` `w` `x`
+        `y` `z` `0` `1` `2` `3` `4` `5` `6` `7` `8` `9` `+` `/`
+    </emu-grammar>
+
+    <emu-note>
```

Comment:
```suggestion
    <emu-note type="editor">
```

Ecmarkup supports "editorial notes", to write notes on how the spec is written rather than on its meaning.

Reply:
Done!